README.md

```diff
@@ -22,7 +22,7 @@ for everyday use. It is aiming for power users with a modern feature mindset.
 - ✅ High-DPI support.
 - ✅ Vertical Line Markers (quickly jump to markers in your history!)
 - ✅ Vi-like input modes for improved selection and copy'n'paste experience and Vi-like `scrolloff` feature.
-- ✅ Blurred behind transparent background when using Windows 10 or KDE window manager on Linux.
+- ✅ Blurred behind transparent background support for Windows 10, the KDE window manager and the GNOME shell on Linux.
```

Comment:
I assume that one Windows 11 it also works, and we can write that we support both KDE and GNOME desktop environments

Reply:
@christianparpart is windows 11 also supported? I could write Windows 10+ in that case

Reply:
Yes, Windows 10 and above. :)